amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/polling/PollingConfig.java

```diff
@@ -44,6 +44,8 @@ public class PollingConfig implements RetrievalSpecificConfig {
 
     public static final int DEFAULT_MAX_RECORDS = 10000;
 
+    public static final long MIN_IDLE_MILLIS_BETWEEN_READS = 200L;
```

Comment:
Don't like having 2 declarations of the same `static final` var, but don't see any existing constants classes.